articles/manual/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Q# Manual | Microsoft Docs 
+description: Q# Manual
+author: E. Gonzalez
+ms.author: v-edsanc@microsoft.com 
+ms.date: 07/31/2019
+ms.topic: article
+uid: microsoft.quantum.manual.intro
+---
+
+# Q# Manual
+
+## What is Q#?
+
+Q# (Q-sharp) is a domain-specific programming language used for
+expressing quantum algorithms.
+It is to be used for writing subroutines that execute on an adjunct
+quantum processor, under the control of a classical host program and computer.
+Until quantum processors are widely available, Q# subroutines execute on a simulator.
+
+A natural model for quantum computation is to treat the quantum computer
+as a coprocessor, similar to that used for GPUs, FPGAs, and other adjunct
+processors.
+The primary control logic runs classical code on a classical "host" computer.
+When appropriate and necessary, the host program can invoke a subroutine
+that runs on the adjunct processor.
+When the subroutine completes, the host program gets access to the
+subroutine's results.
+
+Q# provides a small set of primitive types, along with two ways
+(arrays and tuples) for creating new, structured types.
+It supports a basic procedural model for writing programs,
+with loops and if/then statements.
+The top-level constructs in Q# are user defined types, operations,
+and functions.
+
+Q# is scalable: it can be used to write simple demonstration programs like teleport that run on a few qubits, but also supports writing large, sophisticated programs such as simulations of complex molecules that will require large machines with millions of qubits. Even though large physical machines are still in the future, Q# allows a programmer to program complex quantum algorithms now. What is more, Q# supports various tasks such as debugging, profiling, resource estimation, and certain special-purpose simulations in a scalable way.
+
+## Why Q#?
+
+[![Quantum computing video](http://img.youtube.com/vi/cOUrzxyng04/0.jpg)](http://www.youtube.com/watch?v=cOUrzxyng04 "")
+
+ While quantum hardware is still in early development, we need to build software for quantum computers to take advantage
+ of the quantum power of computation when the time comes. Q# is a tool made to provide a platform where developers can write and test programs to be run on a future generation of quantum computers.
+
+For more information about the foundations and motivation of Q# check out [Why do we need Q#?](https://devblogs.microsoft.com/qsharp/why-do-we-need-q/).
+
+## How to learn Q#?
+
+### Pre-required knowledge to use Q#
+
+There is no need to be an expert on quantum physics to start writing your first quantum programs. However, there are some concepts that are essential to understand quantum computing:
```

Comment:
I think this could read in a way that scares off new readers; there's plenty of room to learn by doing, especially with the katas, books, courses, and so forth that are now available. Especially since you do a great job of listing those below, it might be good to invite the reader to keep reading even if they don't yet have all the background they will eventually need.

Reply:
Mmm, yes, you're right, it can scare the reader (and maybe "Pre-required knowledge" is a bit rough). However, I think there should be somewhere a small list of essential concepts that one should know. For example, Quantum Katas are great but they start assuming the reader is familiar with concepts such as complex numbers, quantum states, qubits, superposition... which a lot of readers might not be familiar with. Stepping into tutorials that explain things in terms of concepts the reader potentially doesn't know could also scare some users. Books don't suffer from this problem, but most of the first contacts with Q# will likely happen through the most accessible tutorials available.

Although I'm not the best person to judge this because I was already familiar with these concepts when I started to go through the katas. Maybe we should consult someone who started the tutorials with a non-physics background to ask what is the optimal way of present it.